examples/basecoin/app/app.go

```diff
@@ -60,6 +63,9 @@ func NewBasecoinApp(logger log.Logger, db dbm.DB) *BasecoinApp {
 		&types.AppAccount{}, // prototype
 	).Seal()
 
+	// Define the feeHandler.
+	app.feeHandler = func(ctx sdk.Context, fees sdk.Coins) {}
```

Comment:
Should be defined below coinKeeper?  Would be nice to have the Tx too.

Reply:
Then the function body would reference coinKeeper as a closure.   Same with democoin.